server/tournaments/index.ts

```diff
@@ -1695,6 +1697,17 @@ export const commands: ChatCommands = {
 					}
 				}
 			}
+		} else if (cmd === 'export') {
+			if (room.game && room.game.gameid === 'tournament') {
+				if (!this.can('gamemoderation', null, room)) return;
```

Comment:
Why do a user need auth to export the tournament? They can already do so manually and it contains no sensitive data,